### PR TITLE
feat: authentication pages and quit quiz button

### PR DESCRIPTION
**Pull request by harshitadaswani**
- added login, signup and forgot password screens
- added quit quiz button in questions screen

### CODE REVIEW

File: screens/forget-password.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+    <script src="https://kit.fontawesome.com/7fa15c8e24.js" crossorigin="anonymous"></script>
+    <title>Forget Password | Crypto Quiz</title>
+</head>
+<body>
+    <div class="pagewrap">
+        <section class="p-m m-m flex flex-column flex-align-center">
+            <h1 class="m-s p-s action-color fw-bold">Forget Password</h1>
+            <div class="rules-container flex flex-column flex-align-center grey-bg p-l m-l br-m">
+                <form action="" method="get" class="p-m m-m br-s">
```
Comment (Manju-sree) [inline screens/forget-password.html]: please don't keep empty attribute action=""

**Review by Bhakti199 — APPROVED** [review]
everything looks fine to me, good to merge

File: screens/login.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+    <script src="https://kit.fontawesome.com/7fa15c8e24.js" crossorigin="anonymous"></script>
+    <title>Login | Crypto Quiz</title>
+</head>
+<body>
+    <div class="pagewrap">
+        <section class="p-m m-m flex flex-column flex-align-center">
+            <h1 class="m-s p-s action-color fw-bold">Login</h1>
+            <div class="rules-container flex flex-column flex-align-center grey-bg p-l m-l br-m">
+                <form action="" method="get" class="p-m m-m br-s">
```
Comment (Manju-sree) [inline screens/login.html]: same here action=""

**Review by Manju-sree — APPROVED** [review]
few changes suggested. Everything else looks good to me.

File: screens/signup.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    <title>Signup | Crypto Quiz</title>
+</head>
+<body>
+    <div class="pagewrap">
+        <section class="p-m m-m flex flex-column flex-align-center">
+            <h1 class="m-s p-s action-color fw-bold">Signup</h1>
+            <div class="rules-container flex flex-column flex-align-center grey-bg p-l m-l br-m">
+                <form action="" method="get" class="p-m m-m br-s">
+                    <div class="flex flex-column">
+                        <label for="Email" class="m-xs">Email Address:</label>
```
Comment (Manju-sree) [inline screens/signup.html:20]: You have used for label for="password" attribute but haven't mentioned the corresponding id to the input element

**Review by t007rushi — APPROVED** [review]
just work on clear naming convention in future

File: screens/forget-password.html (hunk 1 above)
Comment (harshitadaswani) [reply screens/forget-password.html]: done

File: screens/login.html (hunk 2 above)
Comment (harshitadaswani) [reply screens/login.html]: done

File: screens/signup.html (hunk 3 above)
Comment (harshitadaswani) [reply screens/signup.html:20]: mentioned

**Pull request merged**